es/Algoritmos de selección/Selección Rápida.md

```diff
@@ -0,0 +1,51 @@
+# Selección rápida o Quick Select
+
+### Definición del problema
+
+Given an array, find the kth largest / smallest element in linear time complexity.
```

Comment:
This is not being translated to Spanish. Please translate it.

Reply:
Oops!!! My mistake.
I've just fixed it. Thank you... I'll be more careful next time